test/services/schema-object-factory.spec.ts

```diff
@@ -57,6 +59,13 @@ describe('SchemaObjectFactory', () => {
 
       @ApiProperty({ enum: Ranking, enumName: 'Ranking', isArray: true })
       rankings: Ranking[];
+
+      @ApiProperty({
+        enum: getAvailableUserRoles(),
```

Comment:
I think here you need to change this to be

```suggestion
        enum: getAvailableUserRoles,
```

Otherwise the function will be called on import 😄

Reply:
My guess would be that you will also need to therefore change schema-object-factory.

Let me know if I can help

Reply:
Thank you for pointing it out @slukes. I'll let you if I need any help changing it.